packages/wallet/src/W3mFrameHelpers.ts

```diff
@@ -67,6 +72,28 @@ export const W3mFrameHelpers = {
     return (request as { payload: W3mFrameTypes.RPCRequest })?.payload?.method
   },
 
+  getResponseType(response: unknown) {
```

Comment:
Here I'm checking both type and payload and returning a string of possible RPC response types that I defined in `W3mFrameConstants`. This would allow us to use less logics in the client and take actions depending on the response type (error, tx hash, or any other thing)